src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -220,6 +220,9 @@ private void registerItems() {
         registerMagicalPlant("Slime", new ItemStack(Material.SLIME_BALL, 8), "90e65e6e5113a5187dad46dfad3d3bf85e8ef807f82aac228a59c4a95d6f6a",
         new ItemStack[] {null, new ItemStack(Material.SLIME_BALL), null, new ItemStack(Material.SLIME_BALL), getItem("ENDER_PLANT"), new ItemStack(Material.SLIME_BALL), null, new ItemStack(Material.SLIME_BALL), null});
 
+        registerMagicalPlant("Uranium", new CustomItem(SlimefunItems.TINY_URANIUM, 1), "c8b29afa6d6dc923e2e1324bf8192750f7bdbddc689632a2b6c18d9fe7a5e",
+        new ItemStack[] {null, SlimefunItems.URANIUM, null, SlimefunItems.URANIUM, getItem("SLIME_PLANT"), SlimefunItems.URANIUM, null, SlimefunItems.URANIUM, null});
```

Comment:
Not sure if "Slime Plant" is the right base here for such an endgame plant, diamond or emerald seems more reasonable in my opinion.

Reply:
Netherite plant might be more suiting if we talk end game.

Reply:
Diamond/Emerald is quite easy to get in most survival and sky environments so I feel netherite would be a good base for this plant as it would require the nether and obtaining the materials which are quite rare and would match the rarity of Uranium.

Reply:
That would make this plant only obtainable in 1.16+ though

Reply:
Require nether_star for base? Since Uranium is fairly high tier resource?

Reply:
Hmm, sure. Perhaps keep the slime plant then and add a Nether star on top?